src/apps/Settings/Settings.js

```diff
@@ -148,7 +148,7 @@ class Settings extends React.Component {
         app.hasWebApp && filtered.push(app.name)
         return filtered
       },
-      ['Home']
+      ['Use the default']
```

Comment:
Should we put this also in the HomeSettingsComponent?

Reply:
This is solved, on this commit HomeSettingsComponent still do not exists.